* grobian@gentoo.org: the use of nanosleep requires librealtime (-lrt)
                      on Solaris, OpenSolaris however doesn't need it
                      anymore

SABENA: http://thread.gmane.org/gmane.comp.video.ffmpeg.devel/83705

--- configure
+++ configure
@@ -1904,6 +1904,13 @@
     fi
 done
 
+# Solaris has nanosleep in -lrt, OpenSolaris no longer needs that
+if check_func nanosleep; then
+	:
+elif check_func nanosleep -lrt; then
+	add_extralibs -lrt
+fi
+
 check_lib math.h sin -lm
 
 # test for C99 functions in math.h
